tests/integration/cqlengine/columns/test_validation.py

```diff
@@ -88,6 +88,23 @@ def test_datetime_none(self):
         dts = self.DatetimeTest.objects.filter(test_id=1).values_list('created_at')
         assert dts[0][0] is None
 
+    def test_datetime_invalid(self):
+        dt_value= 'INVALID'
+        with self.assertRaises(TypeError):
+            self.DatetimeTest.objects.create(test_id=2, created_at=dt_value)
+
+    def test_datetime_timestamp(self):
+        dt_value = 1454520554
+        self.DatetimeTest.objects.create(test_id=2, created_at=dt_value)
```

Comment:
I don't think this makes a huge difference, but I think this ``test_id`` should be different among tests.